runtime/common/src/lib.rs

```diff
@@ -273,6 +275,22 @@ pub type EnsureRootOrTwoThirdsTechnicalCommittee = EitherOfDiverse<
 	pallet_collective::EnsureProportionAtLeast<AccountId, TechnicalCommitteeInstance, 2, 3>,
 >;
 
+/// Developer Committee
+pub type EnsureRootOrAllDeveloperCommittee = EitherOfDiverse<
+	EnsureRoot<AccountId>,
+	pallet_collective::EnsureProportionAtLeast<AccountId, DeveloperCommitteeInstance, 1, 1>,
+>;
+
+pub type EnsureRootOrHalfDeveloperCommittee = EitherOfDiverse<
+	EnsureRoot<AccountId>,
+	pallet_collective::EnsureProportionAtLeast<AccountId, DeveloperCommitteeInstance, 1, 2>,
+>;
+
+pub type EnsureRootOrTwoThirdsDeveloperCommittee = EitherOfDiverse<
+	EnsureRoot<AccountId>,
+	pallet_collective::EnsureProportionAtLeast<AccountId, DeveloperCommitteeInstance, 2, 3>,
+>;
+
```

Comment:
These are acutally not used, aren't they

Reply:
Removed ✅